website/app/templates/error.hbs

```diff
@@ -0,0 +1,26 @@
+{{page-title "Error"}}
+
+<div class="doc-page-error">
+  <div class="doc-page-error__wrapper">
+    <img
+      class="doc-page-error__image"
+      src="/assets/other/jory-travolta.gif"
+      alt="Animated gif created by Jory Tindall, showing a classic John Travolta meme on top of a composition of HDS component"
+    />
+    <div class="doc-page-error__content">
+      <h1 class="doc-text-h1">Well shoot...</h1>
+      <p class="doc-text-body">We can't seem to find what you're looking for or some unexpected error occurred. Head
+        back to the
+        <LinkTo class="doc-link-generic" @route="index">homepage</LinkTo>.</p>
+      <h4 class="doc-text-h4">Other helpful resources</h4>
+      <ul>
+        <li><LinkTo class="doc-link-generic" @route="about">About</LinkTo></li>
```

Comment:
@didoo do we have any existing styles for styling lists like this?
<img width="323" alt="Screen Shot 2022-12-14 at 11 23 37" src="https://user-images.githubusercontent.com/1672302/207664470-9592b829-6706-4925-ab85-8f4d69ff72d6.png">

Reply:
I intended to create a mixin for sharing this list style between the markdown list and the other normal HTML lists that use a specific class (eg. `doc-list-generic`) similar to what I have already done for the `doc-link-generic`. 
Probably I would suggest to leave it out for now (the styling) and create a JIRA ticket assigned to me, to add the style to it (should be a 5 mins job)

Reply:
https://hashicorp.atlassian.net/browse/HDS-1248